theatre/studio/src/uiComponents/simpleContextMenu/RightClickMenu/RightClickMenu.tsx

```diff
@@ -45,6 +45,7 @@ const RightClickMenu: React.FC<{
   items: IContextMenuItem[] | (() => IContextMenuItem[])
   rightClickPoint: {clientX: number; clientY: number}
   onRequestClose: () => void
+  keepOpen?: boolean
```

Comment:
Change: Add inline doc comment for why `keepOpen` is useful, and what it prevents.

Reply:
I might of missed it, but please double check that this `keepOpen` is actually being used for something.

Reply:
it is dead. Removed. ty.